File: codes/c++/primeno/primeno.cpp
#include<iostream>
using namespace std;

int main() {
    int n, i;
    cout<<"Enter the number: ";
    cin>>n;
    if(n==1) {
        cout<<"The smallest prime number is 2.";
    }
    for(i=2; i<=n; i++) {
        if(n%2==0) {
            cout<<"The number is not prime.";
            break;
        }
        if(n==i) {
            cout<<"The number is prime.";
        }
    }
}

File: codes/java/primeno/Primeno.java
import java.util.Scanner;

class Primeno {
    public static void main(String[] args) {
        int n,i;
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter any number: ");
        n = sc.nextInt();
        sc.close();
        if(n == 1) {
            System.out.println("The smallest prime number is 2.");
        }
        for(i=2; i<=n; i++) {
            if(n%2 == 0) {
                System.out.println("The number is not prime.");
                break;
            }
        }
        if(n == i) {
            System.out.println("The number is prime.");
        }
    }
}